smithy-typescript-codegen/src/main/resources/software/amazon/smithy/typescript/codegen/integration/default_readme_server.md.template

```diff
@@ -0,0 +1,42 @@
+<!-- generated file, do not edit directly -->
+
+# ${packageName}
+
+## Description
+
+JavaScript Server SDK for ${serviceId}
+
+${documentation}
+
+## Installing
+To install this package, simply type add or install ${packageName}
+using your favorite package manager:
+- `npm install ${packageName}`
+- `yarn add ${packageName}`
+- `pnpm add ${packageName}`
+
+## Getting Started
+
+Below is an example service handler created for the ${commandName} operation.
+
+```ts
+
+import {ServiceHandler} from "@aws-smithy/server-common";
+import {${commandName}ServerInput, ${commandName}ServerOutput} from "${packageName}";
+import {convertEvent, convertVersion1Response} from "@aws-smithy/server-apigateway";
+
+const ${commandName}Operation: Operation<${commandName}ServerInput, ${commandName}ServerOutput> = async (input, _) => {
+  // Populate your business logic
+};
+
+const serviceHandler: ServiceHandler<HandlerContext> = get${commandName}Handler(${commandName}Operation);
+```
+
+## Contributing
+
+This client code is generated automatically. Any modifications will be overwritten the next time the `${packageName}` package is updated.
```

Comment:
This says `client`, but either ways, similar to the client, we should remove these sections on Contributing/License.